README.md

```diff
@@ -4,4 +4,9 @@ Module 1:creating Hello Word Exercise
 .A .css file that includes one class with styles for your header (make it in your favorite color).
 .A test.md file with the text "This file should be ignored by git".
 .Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
-.Open a pull request
\ No newline at end of file
+.Open a pull request
```

Comment:
As described in the requirements, document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [ ] As described in the link above your README file is not descriptive. Please update your README to give more elaborate details about the project and how to set it up locally. Use this [Microverse template](https://github.com/microverseinc/readme-template) for your README

- [ ] Please also note that your commit messages should be in **imperative tense and capitalized**. For example:
  - The commit message `'changes made to headline'` should have been `Change headline` or `Update headline` or something like that.
  - `add Heading` should have been `Add heading`

- [ ] Your Pull Request is empty and does not describe the feature(s) you want to merge. Please add a descriptive Pull Request title and summary describing the features.
![pr_m](https://user-images.githubusercontent.com/36542986/133324512-23a40aef-6bb7-4ae1-a9b5-7cc4be1190ed.jpg)